### ISSUE TITLE
Feedback on the lesson

### ISSUE THREAD

**Issue by KatieMFritz** [KatieMFritz/codelab-sinatra-gems-arrays-uninjected-erb#1]
@chrisvfritz - on page 4, I tried using irb in my git bash command line and it just took me to inspect mode ("require 'cat_api'" returned "true"). I had to open Command Prompt with Ruby and Rails for it to work. I would give a little more instruction so people don't have to wonder or break things.

**Comment by KatieMFritz** [comment]
There is also a factual error at the bottom of this page. No such thing as too many cats.

**Comment by KatieMFritz** [comment]
I don't get the instances variable thing on page 5 yet, but I'll give it time.

**Comment by KatieMFritz** [comment]
The last page could use more context for erb. I created cats.erb with html boilerplate and pasted the lesson code between the body tags.

**Comment by KatieMFritz** [comment]
OH MY GOD SO MANY CATS IN HATS

**Comment by KatieMFritz** [comment]
Where I got stuck with CheapShark:
- I didn't get the instance variable implementation in the lesson example, so I forgot about it when it came to the  project. Splitting up the lesson into more pages might help. I still am not confident about what to put in app.rb and what goes in the erb. 
- The CheapShark API gave an error when there was a blank input - I probably wouldn't have solved that on my own.
- I don't think I would have realized I could do <%= game.title %>, etc. on my own. 
- I didn't have my input box in a form. When I did, I wasn't sure what the form action and method should be (none).
